### ISSUE TITLE
[Bug/Question]The tool doesn't seem to ignore keywords in a multi-line description under Scenario

### ISSUE THREAD

**Issue by bloodteller123** [CXwudi/comp5903-easy-cucumber#14]
**Describe the bug or Question**
Tittle

**(For bug report only) To Reproduce**
Given a scenario, where a long description is broken down into multi-line texts
```
Scenario: Row 84 roll 2 skulls, 3 parrots, 3 coins,put 3 coins in chest
    then rerolls 3 parrots and get 2 diamonds 1 coin, put coin in chest (now 4)
    then reroll 2 diamonds and get 1 skull 1 coin, score for chest only = 400 + 200 = 600 AND report death
    Given players are initialized
    And dice are initialized
    When player 1 has fortunate card "chest"
```
The tool then asks me to implement step definition for ```then``` in the description.

**(If applicable) Stack Trace**

<details><summary>logs</summary><p>

Exception in thread "main" scs.comp5903.cucumber.model.exception.EasyCucumberException: EZCU013: Step definition not found for: ThenStep(stepString=reroll 2 diamonds and get 1 skull 1 coin, score for chest only = 400 + 200 = 600 AND report death). Are you sure you implemented this step definition? Or did you forget to make the method public?
	at scs.comp5903.cucumber.builder.JFeatureBuilder.lambda$mapAllStepsToStepDefMethods$0(JFeatureBuilder.java:89)
	at java.base/java.util.Optional.orElseThrow(Optional.java:408)
	at scs.comp5903.cucumber.builder.JFeatureBuilder.mapAllStepsToStepDefMethods(JFeatureBuilder.java:88)
	at scs.comp5903.cucumber.builder.JFeatureBuilder.build(JFeatureBuilder.java:46)
	at scs.comp5903.cucumber.EasyCucumber.build(EasyCucumber.java:147)
	at scs.comp5903.cucumber.EasyCucumber.build(EasyCucumber.java:68)
</p></details>

**(For bug report only) Expected behavior**
The tool should ignore whatever in the description
**Environment (please complete the following information):**
 - OS: [e.g. macOS 13, Windows Pro 11]
 - Java version: [e.g. Java 11, Java 17]
 - Maven/Gradle version: [e.g. maven 3.8.2, gradle 7.5.1]
 - IDE: [e.g. IntelliJ Ultimate 2022.2.2, Eclipse 2022.6]

**Anything else**
Add any other context about the problem or question here.

**Comment by CXwudi** [comment]
Does the official cucumber ignore these two `then` sentences?

**Comment by bloodteller123** [comment]
I believe so. I ran the same test and it passed.

**Comment by CXwudi** [comment]
Yeap, you are right. I confirmed as well, will fix in 0.3.4

**Comment by CXwudi** [comment]
This is fixed, will be release in 0.3.4
